Source/RSP/Interpreter Ops.c

```diff
@@ -547,7 +547,9 @@ void RSP_Vector_VMUDL (void) {
 	for (el = 0; el < 8; el ++ ) {
 		del = EleSpec[RSPOpC.rs].B[el];
 
-		temp.UW = (DWORD)RSP_Vect[RSPOpC.rd].UHW[el] * (DWORD)RSP_Vect[RSPOpC.rt].UHW[del];
+        temp.UW =

```

Comment:
To comply with standard formatting practices.

The following:
```c
        temp.UW = (uint32_t)RSP_Vect[RSPOpC.rd].UHW[el] + (uint32_t)RSP_Vect[RSPOpC.rt].UHW[del];
```
exceeds 80 characters in the same line and can a) be harder to read the whole line b) cause issues on smaller layouts, monitor sizes or page formatting.
That was why I said this:
```c
        temp.UW =
            (uint32_t)RSP_Vect[RSPOpC.rd].UHW[el]
          + (uint32_t)RSP_Vect[RSPOpC.rt].UHW[del];
```

It's not a big deal here; I can have it all in one single line in a few minutes.